ingestclient/plugins/zarr.py

```diff
@@ -19,6 +19,7 @@
 from .chunk import ChunkProcessor, ZYX_ORDER
 import numpy as np
 import zarr
+from s3fs import S3FileSystem
```

Comment:
we should probably conditionally import these, so you don't need s3fs if you're dong a GCS ingest. just a thought!